scapy/layers/tls/crypto/pkcs1.py

```diff
@@ -220,7 +221,7 @@ def pkcs_emsa_pss_encode(M, emBits, h, mgf, sLen):
     rem = 8*emLen - emBits - 8*l # additionnal bits
     andMask = l*b'\x00'
     if rem:
-        j = chr(reduce(lambda x,y: x+y, map(lambda x: 1<<x, range(8-rem))))
+        j = chr(reduce(operator.add, [1 << x for x in range(8 - rem)]))
```

Comment:
You should use `sum` here: `j = sum(1 << x for x in range(8 - rem))`. Also, even with `reduce()`, we could use generators instead of lists.